exo/networking/grpc/grpc_discovery.py

```diff
@@ -34,6 +35,16 @@ def __init__(self, node_id: str, node_port: int, listen_port: int, broadcast_por
         self.broadcast_task = None
         self.listen_task = None
         self.cleanup_task = None
+        self.interfaces = self.get_network_interfaces()
+
+    def get_network_interfaces(self):
+        interfaces = netifaces.interfaces()
+        if psutil.system() == "Darwin":  # macOS
```

Comment:
Make this `if psutil.MACOS:` and remove the comment

Reply:
Beat me to it!